src/Owin.Security.Providers.Salesforce/SalesforceAuthenticationHandler.cs

```diff
@@ -187,7 +187,12 @@ protected override Task ApplyResponseChallengeAsync()
             var state = Options.StateDataFormat.Protect(properties);
 
             var authorizationEndpoint =
-                $"{Options.Endpoints.AuthorizationEndpoint}?response_type={"code"}&client_id={Options.ClientId}&redirect_uri={HttpUtility.UrlEncode(redirectUri)}&display={"page"}&immediate={false}&state={Uri.EscapeDataString(state)}&scope={""}";
+                $"{Options.Endpoints.AuthorizationEndpoint}?response_type={"code"}&client_id={Options.ClientId}&redirect_uri={HttpUtility.UrlEncode(redirectUri)}&display={"page"}&immediate={false}&state={Uri.EscapeDataString(state)}";
+
+            if (Options.Scope.Count > 0)
```

Comment:
Updated PR. I forgot that you can't use null conditional operator quite like that.
I went with `Options.Scope != null && Options.Scope.Count > 0`.
`(Options.Scope?.Count ?? 0) > 0` is also an option but less readable in my opinion.